src/test/java/http/ResponseHandlerTest.java

```diff
@@ -0,0 +1,173 @@
+package http;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ResponseHandlerTest {
+
+    static RequestHandler requestHandler;
+    static ResponseHandler responseHandler;
+
+    @BeforeAll
+    public static void setup() {
+        requestHandler = new RequestHandler();
+        responseHandler = new ResponseHandler();
+    }
+
+
+    @Test
+    public void pageNotFoundTestReturnsA404() throws IOException {
+        String requestParameters = "GET /a_cat HTTP/1.1\r\n";
+        ByteArrayInputStream input = new ByteArrayInputStream(requestParameters.getBytes());
+
+        String request = requestHandler.getRequest(input);
```

Comment:
Refactor: Create a helper method to avoid repetition of these methods

Reply:
Created helper method to remove highlighted repetition